docs/publishers/authorization-grant-tokens.rst

```diff
@@ -0,0 +1,146 @@
+Generating Authorization Grant Tokens
```

Comment:
Ok, I've replaced this with an [intersphinx](http://www.sphinx-doc.org/en/stable/ext/intersphinx.html) link to the section in the h docs, but that link will be broken until https://github.com/hypothesis/h/pull/4423 is merged

This is one we're gonna have to watch out for - the fact that our code is split into separare repos for web service, client, browser extension etc is an implementation detail that the documentation's audience might not want to be bothered with. Or is it? Anyway, we probably want to avoid distributing docs for one audience across multiple sites too much. But when we do have to, intersphinx should be helpful.